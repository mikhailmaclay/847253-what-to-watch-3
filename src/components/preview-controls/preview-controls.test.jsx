// Libraries
import React from 'react';
import renderer from 'react-test-renderer';
// Components
import PreviewControls from './preview-controls';

describe(`<PreviewControls/>`, () => {
  const props = {
    isMuted: false,
    onToggleMuteButtonClick: () => {}
  };

  it(`should render correctly`, () => {
    const result = renderer.create(
        <PreviewControls {...props}/>
    ).toJSON();

    expect(result).toMatchSnapshot();
  });
});
