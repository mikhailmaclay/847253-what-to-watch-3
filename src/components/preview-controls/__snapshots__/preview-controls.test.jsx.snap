// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreviewControls/> should render correctly 1`] = `
<button
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "transparent",
      "border": "none",
      "bottom": "8px",
      "cursor": "pointer",
      "display": "block",
      "margin": "0",
      "padding": "0 4px",
      "pointerEvents": "auto",
      "position": "absolute",
      "right": "8px",
      "transform": "translate3d(0, 0, 0)",
      "zIndex": "3",
    }
  }
  title="Mute"
  type="button"
>
  <svg
    height="15"
    style={
      Object {
        "transform": "",
      }
    }
    viewBox="0 0 15 15"
    width="15"
  >
    <use
      xlinkHref="#not-muted"
    />
  </svg>
</button>
`;
